e2e-tests/portal/cypress/e2e/journeys/payment-officer/utilisation-report-upload/confirmation.spec.js

```diff
@@ -0,0 +1,23 @@
+const { utilisationReportUpload, confirmAndSend, confirmation } = require('../../../pages');
+const MOCK_USERS = require('../../../../fixtures/users');
+const relativeURL = require('../../../relativeURL');
+
+const { BANK1_PAYMENT_OFFICER1 } = MOCK_USERS;
+
+context('Confirm and send', () => {
+  beforeEach(() => {
+    cy.login(BANK1_PAYMENT_OFFICER1);
+    cy.visit(relativeURL('/utilisation-report-upload'));
+
+    utilisationReportUpload.utilisationReportFileInput().attachFile('test-csv.csv');
+    utilisationReportUpload.continueButton().click();
+    confirmAndSend.confirmAndSendButton().click();
+  });
+
+  it('Should route to the login page when the sign-out button is selected', () => {
```

Comment:
we don't properly check we've made it to confirmation page anywhere - can you just add a check title exists style assertion at the start of this test please?